### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by sasik007** [Alex313031/thorium#727] [Q&A]
Hello, is it possible to update automatically via chrlauncher? If so, what should I enter in the ini file?

**Comment by sasik007** [comment]
is this browser still under development? Is the project dead?

**Reply by gz83** [reply]
The project is not "dead". The premise for the project to be updated is that Alex has free time.

**Reply by RoyRao2333** [reply]
Still the discussion talked about before.

The fundamental thing about a project being actively maintained is that it can't be left in the hands of just one person to release versions. Maybe you need to discuss this within your team, so that every collaborator has the power to release a version (assuming that a full code review has been done, especially by Alex).

It's great that Thorium is being used by more people, so all the more reason for you to take it seriously. :)

Just a few personal crumbs, thanks to all you guys. @gz83

**Comment by sasik007** [comment]
ok, thank you for your answer, and can you answer my earlier question regarding this update using chrlauncher? Will there be a possibility of cancer?

**Comment by Alex313031** [comment]
@sasik007 @RoyRao2333 M126 coming soon. M125 was skipped due to being too old. 

And what do you mean by cancer?
The preferred auto updater for Thorium is > https://github.com/Alex313031/thorium-winupdater (I need to make new builds to incorporate the latest changes though)

**Comment by GitAlexToTheGreek** [comment]
Hi there 👋

Just getting started with the browser... 

I notice that there's already a "WinUpdater" folder found at C:\Users\username\AppData\Local\Thorium containing "Thorium-WinUpdater.exe"...

-> Is it necessary to occasionally check for updates by manually clicking that exe or will this process be done automatically?

**Comment by gz83** [comment]
I believe it's still necessary to download the new version via GitHub.
